package.json

```diff
@@ -59,7 +59,7 @@
     "@types/chai-subset": "1.3.3",
     "@types/jest": "29.4.0",
     "@types/jsonwebtoken": "9.0.1",
-    "@types/node": "18.14.0",
+    "@types/node": "18.18.0",
```

Comment:
Unrelated change.

Reply:
Using version `18.14.0`, I have found error about type mismatch on `node-fetch` at https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/blob/e0bc28ebb5a4d0dd04d2d5e66c8610c6d9f13324/feature-runner/lib/b2c/accessToken.ts#L58

The error message is

```
Type 'URLSearchParams' is not assignable to type 'BodyInit'.
  Property 'size' is missing in type 'import("url").URLSearchParams' but required in type 'URLSearchParams'.ts(2322)
lib.dom.d.ts(22575, 14): 'size' is declared here.
index.d.ts(75, 2): The expected type comes from property 'body' which is declared here on type 'RequestInit'
```

Reply:
Get rid of node-fetch and use native fetch (which is available since v18)

Reply:
After removing `node-fetch` and use native fetch, I can then revert `@types/node`changes.